library/junos_install_config

```diff
@@ -250,6 +261,9 @@ def _load_via_netconf(module):
         logging.error(msg)
         module.fail_json(msg=msg)
 
+    if args['ignore_warning'] in BOOLEANS + ["True", "False"]:
```

Comment:
I have not defined ignore_warning as a string for few reasons. With the older version of Ansible type will be taken care of. So if I pass in Boolean, string, list it will remain so.
Do you suggest me to make ignore_warning type defined as string?